python/src/rest_api/__init__.py

```diff
@@ -0,0 +1,90 @@
+"""__init__.py module for the REST API plugin."""
+
+# WARNING - DO NOT EDIT - YOUR CHANGES WILL NOT BE PROTECTED.
+# This file is auto-generated by the aac gen-plugin and may be overwritten.
+
+from copy import deepcopy
+from os.path import join, dirname
+from typing import Any
+
+from aac.context.definition import Definition
+from aac.context.language_context import LanguageContext
+from aac.context.source_location import SourceLocation
+from aac.execute import hookimpl
+from aac.execute.aac_execution_result import ExecutionResult, ExecutionStatus
+from aac.execute.plugin_runner import PluginRunner
+from aac.in_out.files.aac_file import AaCFile
+
+from rest_api.rest_api_impl import plugin_name, rest_api, gen_openapi_spec
+
+
+rest_api_aac_file_name = "rest_api.aac"
+
+
+def run_rest_api(host: str, port: int) -> ExecutionResult:
+    """
+     Start a RESTful interface for interacting with and managing AaC.
+     Args:
+         host (str): Set the hostname of the service. Useful for operating behind proxies.port (int): The port to which the RESTful service will be bound.
+
```

Comment:
```suggestion
         host (str): Set the hostname of the service. Useful for operating behind proxies.
         port (int): The port to which the RESTful service will be bound.

```

Not sure why this didn't go down to a new line, iteration of args was working fine following our tweaks I could have sworn....